.github/CONTRIBUTING.md

```diff
@@ -56,15 +56,29 @@ Adhering to the following process is the best way to get your work included in t
    git pull [--rebase] upstream master
    ```
 
-5. Squash your commits down to a single one (we want to keep the master branch nice and clean)
+5. Set-up the website
 
-5. Push your topic branch up to your fork:
+  ```bash
+  # install dependencies
+  npm install
+  # run the website and watch for changes (http://localhost:9090)
+  npm start
```

Comment:
little bit currection of the css (designing part) .. java script include the website . this is my heartly suggestion